src/EVEMon.Common/Enumerations/CCPAPI/ESIAPICorporationMethods.cs

```diff
@@ -100,66 +112,79 @@ public enum ESIAPICorporationMethods : ulong
         /// <summary>
         /// Corporation member information.
         /// </summary>
+        [ESIMethod("/v1/corporations/{0:D}/membertracking/", Scope = "esi-corporations.track_members.v1")]
         CorporationMemberTracking = 1 << 15,
-        
+
         /// <summary>
         /// List of all outposts controlled by the corporation.
         /// </summary>
+        [ESIMethod("/v1/corporations/{0:D}/outposts/", Scope = "esi-corporations.read_outposts.v1")]
```

Comment:
there is no outpost API
https://esi.evetech.net/ui/?datasource=tranquility&version=latest#/
and now that scope `esi-corporations.read_outposts.v1` not supported

Reply:
On one of my other pull requests https://github.com/peterhaneve/evemon/pull/88 @peterhaneve said that a rewrite of the ESI frontend wasn't in the cards for now, so I am not going to spend time fixing this up until then.